src/components/accordion/accordion.js

```diff
@@ -4,28 +4,34 @@ import classNames from "classnames";
 import ArrowDown from "../../images/icons/arrow-down.svg";
 import "./accordion.scss";
 
-export const Accordion = ({ title, content }) => {
+export const Accordion = ({ title, content, logo }) => {
   const [showContent, setShowContent] = useState(false);
 
   return (
-    <div className="message">
+    <div className="message __accordion">
       <div
         className={classNames("message-header __accordion-header", {
           "__accordion-header--expanded": showContent
         })}
         onClick={() => setShowContent(!showContent)}
       >
-        {title}
+        <span className="__accordion-title">{title}</span>
         <div className="__accordion-arrow">
           <ArrowDown />
         </div>
       </div>
-      {showContent && <div className="message-body">{content}</div>}
+      {showContent && (
+        <div className="message-body">
+          {logo && <img className="__accordion-logo" src={logo} />}
```

Comment:
Hmm, I wouldn't pass the image/logo as a prop, but have it as a child in the `content` prop, or perhaps change the `content` prop to be `children` instead? I think that would more appropriate now that we have more than paragraphs in the content of the accordion.